test/draw/RasterRgbaTest.ooc

```diff
@@ -16,6 +16,15 @@ RasterRgbaTest: class extends Fixture {
 	sourceFlower := "test/draw/input/Flower.png"
 	init: func {
 		super("RasterRgbaTest")
+		this add("Test file not found", func {
+			try {
+				image := RasterRgba open("nonExistingFile")
+				expect(false)
```

Comment:
I guess a comment would be good to add here, that line is there since that code should be unreachable since we are expecting an exception to trigger on the line above. So fi that row is executed the test should fail. (The row below is due to that the compiler says the variable Image is "unused" and I'm not sure to what level the compiler optimizes. I don't want the critical part of the test to be optimized away)